### PR TITLE
initial robots.txt

### PR DESCRIPTION
**Pull request by rosskarchner**
It's a robots.txt file!

We will almost certainly iterate on this, but I think this is an OK start. I've only chosen to disallow URL's that produce PDF's, and the "print" versions of Spanish AskCFPB pages (the english versions were removed in the recent redesign).

I've also used the 'sitemap' declaration to point to the activity log RSS feed, since [the sitemap protocol allows it](http://www.sitemaps.org/protocol.html#otherformats). We should still look into a "real" sitemap, but this should allow search engines to pick up new content faster (at least, content that's reflected in the activity feed)

Once merged (and deployed), we'll need to create an alias from /robots.txt to  (site location on server/static/robots.txt

## Additions

- a robots.txt files

## Review

- @willbarton 
- @sarahsimpson09 
- @schaferjh 

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: cfgov/core/static/robots.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+User-agent: *
+Disallow: /save-hud-counselors-list
+Disallow: /about-us/the-bureau/leadership-calendar/pdf
+Disallow: /es/obtener-respuestas/*.imprimir.html
```
Comment (Scotchester) [inline cfgov/core/static/robots.txt]: Please add: `Disallow: /external-site`

File: cfgov/core/static/robots.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+sitemap: http://www.consumerfinance.gov/activity-log/feed/
```
Comment (Scotchester) [inline cfgov/core/static/robots.txt]: This URL doesn't point to a sitemap.

Comment (Scotchester) [inline cfgov/core/static/robots.txt]: Sorry... didn't read the OP. 😳

Comment (rosskarchner) [inline cfgov/core/static/robots.txt]: the sitemap protocol allows for RSS feeds to stand in: http://www.sitemaps.org/protocol.html#otherformats

File: cfgov/core/static/robots.txt (hunk 1 above)
Comment (rosskarchner) [inline cfgov/core/static/robots.txt]: done!

**Comment by schaferjh** [comment]
Sitemap should happen soon... 👍

**Pull request merged**